diceware/__init__.py

```diff
@@ -208,4 +210,12 @@ def main(args=None):
     if options.version:
         print_version()
         raise SystemExit(0)
-    print(get_passphrase(options))
+    try:
+        print(get_passphrase(options))
+    except (OSError, IOError) as infile_error:
+        if getattr(infile_error, 'errno', 0) == ENOENT:
+            logging.getLogger('ulif.diceware').error(
+                "The file '%s' does not exist." % infile_error.filename)
```

Comment:
It is okay to invent a new error message text here, but not neccessary.

Reply:
Any suggestions for different error message?

Reply:
What about `...eror(infile_error)`?

Reply:
It would be like this then,
`[Errno 2] No such file or directory: 'nonexisting'`
what do you think?

Reply:
Looks good to me 👍